content/_data/changelogs/lts.yml

```diff
@@ -499,6 +499,45 @@
         Fix for <code>NullPointerException</code> while initiating some SSH connections (regression in 2.59).
       issue: 44120
       pull: 2888
+- version: "2.60.2"
+  date: 2017-07-19
+  changes:
+    - type: bug
+      message: >
+        Add documentation for time zone specification for cron patterns (e.g. SCM polling).
+      issue: 9283
+      pull: 2927
+    - type: bug
+      message: >
+        Do not submit form when pressing <kbd>Enter</kbd> in the plugin manager's filter field.
+      pull: 2902
+      issue: 44523
+    - type: bug
+      message: >
+        Jenkins failed to perform some cleanup tasks, including saving the build queue, if stopped via REST <tt>/exit</tt>,
+        CLI <tt>shutdown</tt>, or when restarting from <em>Install as Windows Service</em>.
+      pull: 2908
+      issue: 44589
+    - type: bug
+      message: >
+        Don't check whether disabled administrative monitors are active or not on the <em>Manage Jenkins</em> page.
+      issue: 44608
+      pull: 2909
+    - type: bug
+      message: >
+        When starting the <code>jenkins.war</code> directly, properly check for Java 8 as minimum instead of Java 7 before proceeding.
```

Comment:
Unfortunately https://github.com/jenkinsci/jenkins/pull/2915 hangs :( @amuniz any chance you work on it to get it landed in .3?